pkgs/by-name/sw/sword/package.nix

```diff
@@ -17,12 +24,29 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-QkCc894vrxEIUj4sWsB0XSH57SpceO2HjuncwwNCa4o=";
   };
 
-  nativeBuildInputs = [ pkg-config ];
-  buildInputs = [
-    icu
-    clucene_core
-    curl
-  ];
+  nativeBuildInputs =
+    [
+      pkg-config
+    ]
+    ++ (lib.optionals isWindows [
+      cmake
```

Comment:
Why not to use cmake everywhere?

Reply:
Autotools is the better supported build system for Sword (I'm the upstream maintainer of the CMake toolchain, and I've never been able to achieve 100% parity with the autotools setup). I only maintained it to do cross-builds for Windows rather than needing to maintain MSVC project files.

However, I believe I've cracked the code and almost gotten to the point where I can cross build with autotools now. So I'm fixing up this PR to just use autotools instead of using CMake for Windows.

Reply:
So in the future the idea is to support Autotools for Windows instead of Cmake for Unix?

Reply:
Correct. Autotools is the first class citizen. Supporting Cmake is a "best effort only". And now that the autotools supports cross compiling, there isn't any reason we would need to pursue converting this derivation to Cmake. The only extra feature it offers at the moment is Python and Perl bindings through Swig, which we are not building.